contents/docs/libraries/next-js.md

```diff
@@ -332,25 +335,29 @@ import { PostHog } from 'posthog-node'
 export default function PostHogClient() {
   const posthogClient = new PostHog(process.env.NEXT_PUBLIC_POSTHOG_KEY, {
     host: process.env.NEXT_PUBLIC_POSTHOG_HOST,
+    flushAt: 1,
+    flushInterval: 0
   })
   return posthogClient
 }
 ```
 
-With this component, we can both send events in the body of a component, and get data from PostHog (such as feature flag evaluations) using a server-side function (named `getData()` here).
+> **Note:** Because our server-side `posthog-node` initializations are short-lived, we set `flushAt` to `1` and `flushInterval` to `0`. This ensures that events are sent immediately and not batched. We also need to call `await posthog.shutdownAsync()` after sending events to flush them.
```

Comment:
what does flushAt and flushInterval do? Let's add it

Reply:
updated